src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.html

```diff
@@ -56,7 +55,6 @@ <h2 class="sidebarHeader heading heading3 title">Partner DAO Tokens</h2>
         <token-details
           hide-delete-button.bind="partnerDaoTokens.length === 1 && !isOpenProposalWizard"
           on-delete.call="deleteToken($index, partnerDaoTokens, partnerDAOTokenDetails, stageSettings.partnerDAOTokenDetailsViewModes)"
-          on-saved.call="checkedForUnsavedChanges()"
```

Comment:
Why remove this call altogether?

Reply:
because this check is already done by the validator